### ISSUE TITLE
DropDown with Headers fails 'aria-required-children'

### ISSUE THREAD

**Issue by chadsowald** [microsoft/fluentui#16841]
### Environment Information

- **Package version(s)**: 7.148.2
- **Browser and OS versions**: Chrome (latest) on Win 10

### Describe the issue:
'aria-required-children' is not satisfied for DropDown controls that use Headers.  Notably, DropDowns that do not have headers in them do not have this problem.  Specifically no 'option' items can be found in the case of using Grouping.

### Repro:
1) Go to the DropDown official docs: https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown
2) Open the first example DropDown "Basic uncontrolled example"
3) Run Accessibility Insights FastPass

Expected:
No accessibility issues

Actual:
'aria-required-children' is not satisfied. Specifically no 'option' items can be found:
![image](https://user-images.githubusercontent.com/165767/107016355-b2d1b880-6752-11eb-999b-16e0601ac59b.png)

### Documentation describing expected behavior
https://accessibilityinsights.io/info-examples/web/aria-required-children/

**Comment by chadsowald** [comment]
Perhaps this is related? 

https://github.com/dequelabs/axe-core/issues/2576

**Comment by chadsowald** [comment]
Actually, this looks more relevant  @joschect :

https://github.com/dequelabs/axe-core/issues/2523

**Comment by joschect** [comment]
@chadsowald Thanks so much for doing that research! That really cuts down on time. I believe that you are correct that we are using group correctly to ensure that the header gets read out for its specific children. I agree that this seems to be a problem with Axe rather than fluent. Is that your assessment as well?

**Comment by chadsowald** [comment]
@joschect Agreed!  The generated HTML looks good to me, but I'm no expert :-)  Not sure on your process to track this bug, but feel free to do whatever you need to do with this bug.  I can always point to this for context for others.
